mart/attack/adversary.py

```diff
@@ -114,13 +118,9 @@ def configure_optimizers(self):
     def training_step(self, batch_and_model, batch_idx):
         input, target, model = batch_and_model
```

Comment:
This can be `batch, model = batch_and_model` if you make the change below.

Reply:
[52d3b23](https://github.com/IntelLabs/MART/pull/215/commits/52d3b23174b5e34fb5b18639fb845477e939bae3)